sdk/core/core-client-rest/src/common.ts

```diff
@@ -242,7 +242,13 @@ export type HttpBrowserStreamResponse = HttpResponse & {
  * a raw stream
  */
 export type StreamableMethod<TResponse = PathUncheckedResponse> = PromiseLike<TResponse> & {
+  /**
+   * Returns the response body as a NodeJS stream. Only available in Node-like environments.
+   */
   asNodeStream: () => Promise<HttpNodeStreamResponse>;
+  /**
+   * Returns the response body as a browser stream. Only available in the browser.
+   */
   asBrowserStream: () => Promise<HttpBrowserStreamResponse>;
```

Comment:
Why would we restrict this for example Deno, Bun and others by default I believe support Web Streams.  I think BrowserStream makes it overall confusing and would rather just have this as `asWebStream()` as these are [Web APIs](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API) and not necessarily browser APIs and is noted on the [Node page](https://nodejs.org/api/webstreams.html) as well

Reply:
Sure, we can certainly consider this as a feature in future, and I would support doing that. But in the meantime this fix resolves the immediate issue: the type claims that `asBrowserStream` returns a Web stream in Node-like environments where it actually returns a Node stream.